README.md

```diff
@@ -1,2 +1,198 @@
+- **Personalized product**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Demo Link 🔥](https://moyasiginko.github.io/ToDoList-Project/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need GitHub account and a text editor.
+
+- Node.js
+- npm
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone git@github.com:MoyasiGinko/ToDoList-Project.git
+cd ToDoList-Project
+```
+
+- cd into the project directory
+
+```
+Open the project using a live sever extension that can be found within your code editor extensions.
+```
+
+### Install
+
+- Once you clone the repo, go to the directory and run `npm install` command to get all necessary instalations.
+
+- Build/ watch project (not necessary, but might be needed in case you make some changes in the src folder).
+
+```
+npm run build
+npm run watch
+```
+
+- Run Webpack live server
+
+```
+npm start
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+N/A
+
+### Run tests
+
+```
+npm run test
+```
+
+- ...in your bash (assuming you are in the root of this project);
+
+### Deployment
+
+You can deploy this project using: webpack in your local machine.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👷‍♂️ Author <a name="authors"></a>
+
+👤 **Moyasi**
+
+- GitHub: [@MoyasiGinko](https://github.com/MoyasiGinko)
+- Twitter: [@MoyasiGinko](https://twitter.com/moyasi_ginko)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mahmudur-rahman-a8a151257)
+
+👤 **Sylvester WAMAYA**
+
+- GitHub: [@githubhandle](https://github.com/sylvester-wamaya)
+- Twitter: [@twitterhandle](https://twitter.com/The_Seal_)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sylvester-wamaya-b11a93112/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[interactive functions]**
+- [ ] **[Footer]**
+- [ ] **[Profile picture]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you get inspired by this project you can give me a star 🙌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] kindly consider customizing your readme so that it related to the current project you are working on while doing so consider checking on each section and make sure it aligns with this leaderboard project, not todo list also remember the author's section to contain your data only as its a solo project doing so will make your readme look amazing and professional